src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1542,7 +1542,22 @@ void Convolution::redefineOutputMemory(const std::vector<VectorDims> &newOutputS
 
 MemoryDescPtr Convolution::getSumMemDesc(const primitive_desc &primitive_desc_it) {
     if (getOutputShapeAtPort(0).isDynamic()) {
-        return DnnlExtensionUtils::makeUndefinedDesc(primitive_desc_it.dst_desc(0), getOutputShapeAtPort(0));
+        // When we set input shape with range dims, real input shape maybe mismatch with output shape, we just change
```

Comment:
In fact Conv + Sum optimization assumes that `Output` and `Sum` ports have the same real shapes: only such configuration is supported by oneDNN. All other cases are covered by fallback logic (much slower from infer time persepctive) which in fact exists only because of undefined shapes on Conv+Sum fusing stage. 
This particular example shows that we have ranged shape on `data` path (128 - 256) and  (1) on `sum` path. Such information gives us determined understanding that fused Conv+Sum will be executed via fallback logic. Given that best solution from performance perspective is to disable fusing (`FuseConvolutionSumAndConvolutionSumActivation ` transforamtion) in case ranges give clear knowledge of non-eqaul shapes.

Reply:
Yes, you are right. But I can't get ranged shape to compare in the function `FuseConvolutionSumAndConvolutionSumActivation`.

I had a test in function `FuseConvolutionSumAndConvolutionSumActivation`:
```
auto shape1 = sumNode->getInputShapeAtPort(0);
auto shape2 = sumNode->getInputShapeAtPort(1);
```
Result of shape1 and shap2:
```
shape1=(1.160.18446744073709551615.18446744073709551615)
shape2=(1.160.1.1)
```

Reply:
CPU Shape [contains ](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/cpu_shape.h#L216-L217) `minDims` and `maxDims` fields which represent actual range. You should check their content.
I would also recommend to reuse `Shape::isCompatible` method to figure out shape2 is not compatible with shape1 (represents broadcasting case)